examples/macro/g3libs.C

```diff
@@ -19,15 +19,17 @@
 
 #endif
 
-Bool_t isLibrary(const char* libName)
-{
-/// Helper function which testes the existence of the given library
-/// \param libName  The library name
-
-  if (TString(gSystem->DynamicPathName(libName, kTRUE)) != TString(""))
-    return kTRUE;
-  else
-    return kFALSE;
+namespace g3libutilities {
```

Comment:
please, rename g3libUtilities

Reply:
Ok, as you like. Honestly, I was relying on what we do in O2 here so that's why I used only small letters.